projects/ui-components/package.json

```diff
@@ -13,7 +17,12 @@
     "@angular/platform-browser": "~12.1.1",
     "@laserfiche/lf-js-utils": "3.0.5",
     "rxjs": "~6.6.0",
-    "zone.js": "~0.11.4"
+    "zone.js": "~0.11.4",
```

Comment:
check peerDependencies

Reply:
let us keep the dependencies of lf-metadata here, as npm automatically install the peer dependencies so the users won't have to do it themselves. and if the users don't use lf-metadata in their application, install the dependencies makes no change to the final bundle size of the users' application.